src/DataFixtures/ORM/LoadIdeaData.php

```diff
@@ -171,6 +171,7 @@ public function getDependencies(): array
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`